### PR TITLE
Fix battery voltage measurement

### PR DESCRIPTION
**Pull request by Kameeno**

### CODE REVIEW

File: src/core/CoolBoardSensors.cpp [hunk 1]
```diff
@@ -192,8 +192,8 @@ float CoolBoardSensors::readVBat() {
   // read battery voltage
   int raw = analogRead(A0);
   // convert it to approx. correct tension in volts
-  float val = 6.04 / 1024 * raw;
-
+  float val = 5.47 / 1023.0 * raw;
```
Comment (3rwww1) [inline src/core/CoolBoardSensors.cpp]: Why ? Those are magic numbers. Why 6.04 ? Why 1023 ? Why both ?

**Review by 3rwww1 — DISMISSED** [review]

**Review by LeSuedois — DISMISSED** [review]
The right value is 5.6 after verification.
In theory the right value is 5.7 (10k + 47k)

File: src/core/CoolBoardSensors.cpp (hunk 1 above)
Comment (LeSuedois) [reply src/core/CoolBoardSensors.cpp]: ah and divide by 1024!

**Comment by 3rwww1** [comment]
We are settled for:
```cpp
#define ADC_MAX 1023
#define VOLTAGE_MAX 5.6

val = raw * (VOLTAGE_MAX / ADC_MAX);
```

**Review by 3rwww1 — APPROVED** [review]

**Pull request merged**